doc/Requirements Specification/Render.sh

```diff
@@ -5,6 +5,9 @@ file="Requirements Specification"
 # fail the script if any command fails
 set -e 
 
+# Delete old rendered file
+rm "Requirements Specification.pdf"
```

Comment:
Plus, it fails, if `Requirement Specifications.pdf` doesn’t exist, as you can see in your failing Travis build.